libsolidity/AST.h

```diff
@@ -424,7 +424,7 @@ class FunctionDefinition: public Declaration, public VariableScope, public Docum
 	/// @returns the canonical signature of the function
```

Comment:
Please update the docstring and I think we should keep the word "signature" as part of this function name.